pages/sign-in/kakao/index.page.tsx

```diff
@@ -7,33 +7,34 @@ import { instance } from "@/apis/axios";
 import { PAGE_PATH } from "@/constants/pagePath";
 
 interface KakaoProps {
-  isFirst: boolean;
+  id: number;
+  isComplete: boolean;
 }
 
 export async function getServerSideProps(
   context: GetServerSidePropsContext,
 ): Promise<GetServerSidePropsResult<KakaoProps>> {
-  const code = context.query["code"];
+  const { code } = context.query;
 
   const res = await instance.get(`/login/oauth/kakao?code=${code}`);
-  const { isFirst } = res.data;
+  const { id, isComplete } = res.data;
 
   return {
-    props: { isFirst: Boolean(isFirst) },
+    props: { id, isComplete },
   };
 }
 
-export default function Kakao({ isFirst }: KakaoProps) {
+export default function Kakao({ id, isComplete }: KakaoProps) {
   const router = useRouter();
 
   useEffect(() => {
-    if (isFirst) {
-      router.push(PAGE_PATH.PROFILE_SETUP);
-      return; // TODO: 우혁님 PR merge 된 후에, 상수로 수정 예정
+    if (isComplete === false) {
```

Comment:
```suggestion
    if (!isComplete) {
```
p4: `isComplete` 자체가 `boolean`이라서 이렇게 적어줘도 좋을 것 같아요

Reply:
p4: 여기서 isComplete이 의미하는게 첫 로그인(닉네임같은거 등록한적이 없는상태) 인가요? 그렇다면 네이밍 바꾸는게 좋지않을까 싶습니다

Reply:
백엔드 측에서 넘겨주는 코드를 구조분해할당으로 그대로 받아서 사용하다보니 이렇게 된 것 같습니다😢 변수에 할당해서 이름 한 번 바꿔가지구 사용해보겠습니다!

Reply:
@jae6269 이렇게 나눠서 작성하는 것보다는 `구조분해할당으로 한 번에 작성하는게 훨씬 깔끔해보이기는 해서, 이 부분은 백엔드 측에 요청해서 네이밍을 변경해보는 쪽으로 해보겠습니다...!

```tsx
  const userId = res.data.id;
  const isMember = res.data.isComplete;
```